utils/utils.hpp

```diff
@@ -109,6 +109,20 @@ class Utils
                           const bool zipFile = false,
                           const std::string pathForExtraction = "./../data/")
   {
+    size_t last = downloadPath.find_last_of("/");
```

Comment:
I am not sure if this would work for windows too as that uses a backward slash " \ " instead of forward slash